Cargo.toml

```diff
@@ -26,8 +25,13 @@ traitobject = "0.0.1"
 typeable = "0.1"
 solicit = "0.2"
```

Comment:
As I mentioned in the comments over at #542, you should bump up this version to "0.3" to make sure that hyper doesn't end up requiring openssl transitively through solicit.